source/iterators.tex

```diff
@@ -409,46 +410,55 @@
+  template<class T, size_t N> constexpr reverse_iterator<T*> rbegin(T (&array)[N])  // freestanding
+  template<class T, size_t N> constexpr reverse_iterator<T*> rend(T (&array)[N]);   // freestanding
+  template<class E> constexpr reverse_iterator<const E*>
+    rbegin(initializer_list<E> il);                                                 // freestanding
+  template<class E> constexpr reverse_iterator<const E*>
+    rend(initializer_list<E> il);                                                   // freestanding
+  template<class C> constexpr auto
+    crbegin(const C& c) -> decltype(std::rbegin(c));                                // freestanding
+  template<class C> constexpr auto
+    crend(const C& c) -> decltype(std::rend(c));                                    // freestanding
+
+  template<class C> constexpr auto size(const C& c) -> decltype(c.size());          // freestanding
+  template<class T, size_t N> constexpr size_t
+    size(const T (&array)[N]) noexcept;                                             // freestanding
+  template<class C> constexpr auto ssize(const C& c)                                // freestanding
     -> common_type_t<ptrdiff_t, make_signed_t<decltype(c.size())>>;
-  template<class T, ptrdiff_t N> constexpr ptrdiff_t ssize(const T (&array)[N]) noexcept;
-  template<class C> [[nodiscard]] constexpr auto empty(const C& c) -> decltype(c.empty());
-  template<class T, size_t N> [[nodiscard]] constexpr bool empty(const T (&array)[N]) noexcept;
-  template<class E> [[nodiscard]] constexpr bool empty(initializer_list<E> il) noexcept;
-  template<class C> constexpr auto data(C& c) -> decltype(c.data());
-  template<class C> constexpr auto data(const C& c) -> decltype(c.data());
-  template<class T, size_t N> constexpr T* data(T (&array)[N]) noexcept;
-  template<class E> constexpr const E* data(initializer_list<E> il) noexcept;
+  template<class T, ptrdiff_t N> constexpr ptrdiff_t
+    ssize(const T (&array)[N]) noexcept;                                            // freestanding
+  template<class C> [[nodiscard]] constexpr auto
+    empty(const C& c) -> decltype(c.empty());                                       // freestanding
+  template<class T, size_t N> [[nodiscard]] constexpr bool
+    empty(const T (&array)[N]) noexcept;                                            // freestanding
+  template<class E> [[nodiscard]] constexpr bool
+    empty(initializer_list<E> il) noexcept;                                         // freestanding
+  template<class C> constexpr auto data(C& c) -> decltype(c.data());                // freestanding
+  template<class C> constexpr auto data(const C& c) -> decltype(c.data());          // freestanding
+  template<class T, size_t N> constexpr T* data(T (&array)[N]) noexcept;            // freestanding
+  template<class E> constexpr const E* data(initializer_list<E> il) noexcept;       // freestanding
```

Comment:
The build is failing with
`Overfull \vbox (1.47272pt too high) has occurred while \output is active 
`
The error goes away if you delete this line. Why?

Reply:
It's an overfull `\vbox`, not an `\hbox`. So, deleting one line reduces the amount of vertical space required, and life is good again.

"Why", I can' say. This should not happen.

Reply:
Fixed by bedf8ccbc1c5fd2ee696525416b8d9d34835d9b0 (which updates memoir by about three years).

A table was page-broken differently.